### PR TITLE
GUI Event Bubbling

### PR DESCRIPTION
**Pull request by royibernthal**

### CODE REVIEW

File: content/classes/3.0/EventState.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ If the callback of a given [Observer](/classes/3.0/Observer) set skipNextObserve
 ---|---|---|---
  | mask | number |   
 optional | skipNextObservers | boolean |   
+optional | target | any | The object that originally notified the event
```
Comment (deltakosh) [inline content/classes/3.0/EventState.md]: no need to update the classes doc
it is automatically generated from code comments

Comment (royibernthal) [reply content/classes/3.0/EventState.md]: Oh got it. Is the GUI doc update fine?

Comment (deltakosh) [reply content/classes/3.0/EventState.md]: Yes sir:)

Comment (royibernthal) [reply content/classes/3.0/EventState.md]: Great :)

Comment (deltakosh) [reply content/classes/3.0/EventState.md]: Can you just undo the changes on classes?

Comment (royibernthal) [reply content/classes/3.0/EventState.md]: Done

**Pull request merged**